SW_Output.c

```diff
@@ -357,8 +357,10 @@ static void sumof_wth(SW_WEATHER *v, SW_WEATHER_OUTPUTS *s, OutKey k)
 		s->temp_max += v->now.temp_max[Today];
 		s->temp_min += v->now.temp_min[Today];
 		s->temp_avg += v->now.temp_avg[Today];
-		//added surfaceTemp for sum
-		s->surfaceTemp += v->surfaceTemp;
+		//added surfaceAvg for sum
+        s->surfaceAvg = v->surfaceAvg;

```

Comment:
Could you please also add documentation of the new output variables to the `testing/Input/outsetup.in` file?